Bridge Load Balancing Policies (#144)
scylladb-zpp-2025-csharp-rs-driver/csharp-driver: docs/source/migration-guide/index.md

- Warning: `Rust: MM/dd/yyyy H:mm:ss.fff zzz WARNING [csharp_wrapper::logging] This is a warning message`
- Info: `Rust: MM/dd/yyyy H:mm:ss.fff zzz INFO [csharp_wrapper::logging] This is an info message`
- Verbose: `Rust: MM/dd/yyyy H:mm:ss.fff zzz VERBOSE [csharp_wrapper::logging] This is a verbose message`

## Load Balancing Policies

### DefaultLoadBalancingPolicy

> - DC awarness by default, with DC of the contact point as the local DC was removed. Now, if the user does not specify a Load Balancing Policy when building the `Cluster`, the driver will be DC unaware, treating all nodes equally.

Reviewer: ```suggestion
- DC-awareness-by-default, with DC of the contact point as the local DC, was removed. Now, if the user does not specify a Load Balancing Policy when building the `Cluster`, the driver will be DC unaware, treating all nodes equally (w.r.t. their DCs).
```

### DCAwareRoundRobinPolicy
- The local DC detection mechanism was fully removed, now the local DC must be specified in the constructor

> - `usedHostsPerRemoteDC` field is not supported anymore, there is a new boolean property `PermitDCFailover`. Setting it to true, permits the driver to query nodes from remote DCs, nodes from local dc are always tried first.

Reviewer: ```suggestion
- `usedHostsPerRemoteDC` field is not supported anymore, there is a new boolean property `PermitDCFailover`. Setting it to true permits the driver to query nodes from remote DCs; nodes from local DC are always tried first.
```

- This policy does not use round robin algorithm anymore
The nodes are chosen at random from the nodes from the local 

> DC, exactly like in the Rust driver

Reviewer: ```suggestion
- This policy does not use round robin algorithm anymore.
The nodes are chosen at random from the nodes from the local 
DC, then at random from remote DCs, exactly like in the Rust driver.
```

> ### RoundRobinLoadBalancingPolicy

Reviewer: 🔧 Missing newline.

- This policy does not use round robin algorithm anymore

> The nodes are chosen at random, exactly like in the Rust driver

Reviewer: 🔧 Missing periods.

### NewQueryPlan()

> All the `NewQueryPlan` methods in the policy classes, are no longer supported, since all query routing is handled by the Rust driver.

Reviewer: ```suggestion
### NewQueryPlan()
All the `NewQueryPlan` methods in the policy classes are no longer supported, since all query routing is handled by the Rust driver.
```
